README.md

```diff
@@ -1,20 +1,16 @@
-# Mitiq
+# <a href="https://github.com/unitaryfund/mitiq"><img src="https://github.com/unitaryfund/mitiq/blob/master/docs/source/img/mitiq-logo.png?raw=true" alt="Mitiq logo" width="350"/></a>
```

Comment:
```suggestion
<p align="center">
# <a href="https://github.com/unitaryfund/mitiq"><img src="https://github.com/unitaryfund/mitiq/blob/master/docs/source/img/mitiq-logo.png?raw=true" alt="Mitiq logo" width="350"/></a></p>
```

FWIW if you want to center the image

Reply:
When its centered I see "#" next to the logo, so I'll keep it left-aligned.